### DISCUSSION TITLE
Any chance to have something similar for Amazon?

### DISCUSSION THREAD

**Discussion by AndrewRCD** [rrirower/DP2SceneGraphWizard#118] [Q&A]
Hi
from the Amazon Creator Portal : 
"Starting Oct 31, 2024, we will no longer be accepting submissions for any new Creator apps."
Any chance to have something similar to the DP2SceneGraphWizard but for Amazon Channels? 
Thanks

**Comment by rrirower** [comment]
Possibly, but not any time soon. It will take time to design something using their equivalent of BrightScript.
